middleware/loginValidation.js

```diff
@@ -0,0 +1,13 @@
+const { isEmpty } = require('lodash');
+
+const loginValidation = (req, res, next) => {
+    const { email, password } = req.body;
+    if (isEmpty(email) || isEmpty(password)) {
+        return res.status(500).json({
+            error: 'Body is not full.',
```

Comment:
you can write error  that you can show to customer directly
ex: error: "Invalid input" to avoid error check on the frontend